config/rabe-fs-backup.timer

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Run RaBe Backup Service timer daily
+
+[Timer]
+# Run once per day
+OnCalendar=daily
+# Start immediately if last start was missed (ie when machine was down)
+Persistent=true
+# Delay timer randomly to spread out load (centos-7.3 and up only)
+RandomizedDelaySec=30min
+# Do not coalesce with other timers, no need to save power during a backup
+AccuracySec=1us
```

Comment:
The only effect this has is that the timer just runs when it's needed and does not try to save power by coalescing with other timers. It won't run at an accuracy of 1us sec anyway, that accuracy is governed elsewhere in systemd. I don't see a point in saving CPU wake up cycles when what we want to do is run a script that uses quite some io anyway.